packages/flet/lib/src/controls/navigation_rail.dart

```diff
@@ -154,6 +154,9 @@ class _NavigationRailControlState extends State<NavigationRailControl>
                             parentAdaptive: widget.parentAdaptive)
                         : Text(label));
               }).toList());
+
+          return constrainedControl(
```

Comment:
If it's just about `rtl` property then `baseControl()` can be used instead. The same for others.

Reply:
That's right!